### DISCUSSION TITLE
How do you use input slots to create a custom file input in Vue 3?

### DISCUSSION THREAD

**Discussion by MarcVegas** [jgile/vue-csv-import#120] [Q&A]
Okay, I know somebody already asked this question but there so little detail was provided that it's hard to determine how he got it to work since not much code was provided

I need to create to pass a customized file input element inside `<vue-csv-input v-slot="{file, change}"></vue-csv-input>` but it can't seem to get my selected file from my custom file input. I added the necessary fields objects in the vue-csv-import component and my csv file is not an issue in this case since it works with the default file input provided by the package

So far I've done the following but nothing works as of yet

```
<vue-csv-input v-slot="{file, change}">
     <label for="finput" class="fancy-upload-btn">Upload CSV File</label>
     <input @change="change" type="file" name="file" id="finput">
</vue-csv-input>
```
I also tried including ref="csvRef" to my file input but no luck there also. I could really use some help with this.

**Comment by VanstraelenMickael** [comment]
Answer still needed.
It seems the change function doesn't work ? 

```
<vue-csv-input v-slot="{ file, change }">
      <input type="file" @change="change">
</vue-csv-input>
```
This should work.
I tried the csvRef too, didn't change anything
